src/main/java/com/coreos/jetcd/internal/impl/ClientUtil.java

```diff
@@ -63,4 +66,36 @@ static URI endpointToUri(String endpoint) {
         .nameResolverFactory(factory)
         .usePlaintext(true);
   }
+
+  public static boolean isValidEndpointFormat(String endpoint) {
+    if (endpoint.startsWith("http")) {
+      return isValidURL(endpoint);
+    }
+
+    String[] hostAndPort = endpoint.split(":");
+
+    if (hostAndPort.length != 2) {
+      return false;
+    }
+
+    String host = hostAndPort[0];
+    String port = hostAndPort[1];
+
+    return (isValidHost(host) && isValidPort(port));
+  }
+
+  private static boolean isValidURL(String url) {
+    String[] scheme = {"http"};
+    UrlValidator urlValidator = new UrlValidator(scheme, UrlValidator.ALLOW_LOCAL_URLS);
```

Comment:
Can't change default schemes and options, if use `UrlValidator .getInstance()` method.
Tell me if I misunderstand something.